pybrain/unsupervised/trainers/deepbelief.py

```diff
@@ -110,7 +110,7 @@ def train(self):
             piecenet.sortModules()
 
             dataset = UnsupervisedDataSet(rbm.hiddenDim)
-            for sample, in self.dataset:
+            for sample,somevar in self.dataset:
```

Comment:
Yes but somehow it was not supported by my python version. I dont know if its just my problem or a problem based on python compatibility